tests/vis/image_test.m

```diff
@@ -0,0 +1,178 @@
+% vis.image test
```

Comment:
There's already an test for vis.image on the dev branch which is rigorous enough and doesn't required any extra resources (I use MATLAB's stock images): https://github.com/cortex-lab/signals/blob/dev/tests/image_test.m

Perhaps you missed it?

Reply:
Ah yes